otterdog/eclipse-tycho.jsonnet

```diff
@@ -87,6 +87,23 @@ orgs.newOrg('eclipse-tycho') {
           requires_last_push_approval: false,
           requires_review_thread_resolution: false,
         },
+        orgs.newRepoRuleset('tycho-4.0.x') {
```

Comment:
this ruleset is identical to the one for the main branch, you could just give the ruleset a more common name and add the tycho-4.0.x branch to the include_refs to avoid duplicating configuration.

Reply:
@netomi thanks for the quick reply, I hope I did it correctly now?